Ruth/Chapter_1/exercise_1.12.1

```diff
@@ -0,0 +1,8 @@
+This is my exercise
```

Comment:
@RuthAtem, what is happening here? Why is the content of this file different? Is this exercise_1.12.1? I doubt, have a look at it!